terraform/central-account/variables.tf

```diff
@@ -245,3 +245,8 @@ variable "user_facing_url" {
   type    = string
   default = ""
 }
+
+variable "consoleme_repo" {
+  type    = string
+  default = "https://github.com/Netflix/consoleme/"
```

Comment:
I'm thinking of if this code ends up in production for someone, it might caught them unprepared if we push some breaking change to master. We might want to leave a small note saying it'd be prudent to fix a release before moving to production? 😛 Maybe as a commented configuration in .tfvars

Reply:
Agreed, this is probably a good note to put in documentation.

Reply:
Added docs in the terraform.tfvars example and a comment in the README. Please revisit.